benchmark/README.md

```diff
@@ -67,11 +67,10 @@ TPCH timing results is written to stdout in the following form: `TPCH_Result,<la
 1. Upload [run_python_benchmark.sh](run_python_benchmark.sh) and all [python tpch benchmark](python/) files to the cluster.
 2. Install pyarrow and pandas on all nodes in the cluster. For example, if you are using Conda, you can use the following commands to install them.
     ```shell
-    export PATH=$PATH:/usr/bin/anaconda/bin
-    sudo chown -R sshuser /usr/bin/anaconda
-    conda update --all
-    conda install pandas
-    conda install pyarrow
+    export PATH=$PATH:/path/to/conda    
+    sudo /path/to/conda update --all
+    sudo env "PATH=$PATH" conda install pandas

```

Comment:
don't we want to do `/path/to/conda` everywhere?

Reply:
Agreed, I would put `/path/to/conda` everywhere.  and remove the export command.

Reply:
Cool, I'll make that change